Doc/library/tokenize.rst

```diff
@@ -17,7 +17,7 @@ as well, making it useful for implementing "pretty-printers," including
 colorizers for on-screen displays.
 
 To simplify token stream handling, all :ref:`operators` and :ref:`delimiters`
-tokens are returned using the generic :data:`~token.OP` token type.  The exact
+tokens and Ellipsis are returned using the generic :data:`~token.OP` token type.  The exact
```

Comment:
Ellipsis -> :data:\`Ellipsis\`

Reply:
Still needed

Reply:
I don't understand, what is still needed.

Reply:
Replace Ellipsis with :data:\`Ellipsis\`.